### ISSUE TITLE
WAVE tool reports 'Missing form label' on a collections page

### ISSUE THREAD

**Issue by h-parekh** [samvera-labs/nurax-pre2023#223]
### Descriptive summary
Hyrax Version: 2.1.0.beta1 
Hyrax Branch: v2.1.0.beta1_qa 
Hyrax Revision: 8fe35f54e067fd499a4467b9ccbefa48233d43a2

If I click a specific collection, the 'Search subcollections and works in this collection' box complains about 'Missing form label'. 
This is non-compliance with section 508 as well as WCAG standards.

#### What It Means
A form control does not have a corresponding label.
#### Why It Matters
If a form control does not have a properly associated text label, the function or purpose of that form control may not be presented to screen reader users. Form labels also provide visible descriptions and larger clickable targets for form controls.
#### How to Fix It
If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

### Expected behavior
WAVE tool should not report this error

### Actual behavior
WAVE tool reports an error

### Steps to reproduce the behavior
1. Install the WAVE [extension for chrome](https://chrome.google.com/webstore/detail/wave-evaluation-tool/jbbplnpkjmmeebjpijfedlgcdilocofh)
2.  Load homepage
3. click one of the records displayed in the explore collections tab. You should be on URL: https://<hostname>/collections/<collection_id>
4. Click on the WAVE tool's icon and you will see the error on the 'Details' tab of WAVE tool

NOTE: You can reproduce this on Nurax as well as vanilla hyrax, but there will be other accessibility errors in Nuraax that have been fixed in Vanilla Hyrax.
